config/initializers/debug_partial_template_paths.rb

```diff
@@ -0,0 +1,11 @@
+if Rails.env.production?
```

Comment:
Why do we need to check if the environment is production? Shouldn't it be enough with the environment variable.

Reply:
This is so that we can set the config value to true by default on non-production environments.

Reply:
@pmanrubia addressed in https://github.com/alphagov/smart-answers/pull/3295/commits/b04513ed4e9392b294f8911f8c95225a2a96a548